VATSpy.dat

```diff
@@ -17780,16 +17780,22 @@ EDGG-P|Langen Radar (Pad High) - Langen|EDGG_H|EDGG-P
 EDGG-P|Langen Radar (Pad High) - Langen|EDGG_P|EDGG-P
 EDGG-P|Langen Radar (Pad High) - Langen|EDGG_W|EDGG-P
 EDGG-P|Langen Radar (Pad High) - Langen|EDGG_NW|EDGG-P
-EDMM|Muenchen|EDMM_C|EDMM
-EDMM|Muenchen|EDMM_E|EDMM
-EDMM|Muenchen|EDMM_F|EDMM
-EDMM|Muenchen|EDMM_K|EDMM
-EDMM|Muenchen|EDMM_R|EDMM
-EDMM-G|Muenchen ACC (Gera) - Muenchen|EDMM_G|EDMM-G
-EDMM-G|Muenchen ACC (Gera) - Muenchen|EDMM_A|EDMM-G
-EDMM-G|Muenchen ACC (Gera) - Muenchen|EDMM_T|EDMM-G
-EDMM-R|Muenchen ACC (Roding) - Muenchen|EDMM_S|EDMM-R
+EDMM|Muenchen|EDMM_CTR|EDMM
+EDMM-Z|Muenchen ACC (Zugspitze) - Muenchen|EDMM_Z|EDMM-Z
+EDMM-Z|Muenchen ACC (Zugspitze) - Muenchen|EDMM_E|EDMM-Z
+EDMM-Z|Muenchen ACC (Zugspitze) - Muenchen|EDMM_U|EDMM-Z
+EDMM-Z|Muenchen ACC (Zugspitze) - Muenchen|EDMM_T|EDMM-Z
+EDMM-Z|Muenchen ACC (Zugspitze) - Muenchen|EDMM_N|EDMM-Z
+EDMM-Z|Muenchen ACC (Zugspitze) - Muenchen|EDMM_W|EDMM-Z
+EDMM-Z|Muenchen ACC (Zugspitze) - Muenchen|EDMM_A|EDMM-Z
+EDMM-Z-FL165A|Muenchen ACC (Zugspitze above FL165) - Muenchen|EDMM_Z|EDMM-Z-FL165A
+EDMM-Z-FL165A|Muenchen ACC (Zugspitze above FL165) - Muenchen|EDMM_U|EDMM-Z-FL165A
+EDMM-Z-FL165A|Muenchen ACC (Zugspitze above FL165) - Muenchen|EDMM_T|EDMM-Z-FL165A
+EDMM-Z-FL165A|Muenchen ACC (Zugspitze above FL165) - Muenchen|EDMM_R|EDMM-Z-FL165A

```

Comment:
Just mentioning, VATSpy currently only reads upto the first underscore, so it will not see the diffrence between EDMM_CTR, EDMM_Z_CTR, EDMM_U_CTR etc. None of these will trigger "EDMM-Z-FL165A" to light up.

Reply:
We are aware of that. But on the map pilots can see the different sector parts.